lib/server.rb

```diff
@@ -0,0 +1,43 @@
+require 'data_mapper'
+require 'sinatra'
+require_relative 'user'
+require_relative 'peep'
+require './lib/helpers/application'
+
+env = ENV['RACK_ENV'] || 'development'
+
+# DataMapper::Logger.new($stdout, :debug)
```

Comment:
avoid leaving comments in like this - commit them and then you'll have them in your github history - you don't want dead code in things you submit to github